backend/FwLite/FwLiteMaui/Platforms/Windows/AppUpdateService.cs

```diff
@@ -118,6 +116,7 @@ private async Task<UpdateResult> ApplyUpdate(FwLiteRelease latestRelease, bool q
             });
         asyncOperation.Progress = (info, progressInfo) =>
         {
+            NotifyInstallProgress(progressInfo.percentage, latestRelease);
```

Comment:
❓ Is there a reason to notify before the `progressInfo.state == DeploymentProgressState.Queued` case is handled? I assume that makes 0% available to whatever receives this notification, whether or not it uses it.